crates/telio-traversal/src/endpoint_providers/upnp.rs

```diff
@@ -521,17 +527,18 @@ impl<Wg: WireGuard, I: UpnpEpCommands, E: Backoff> State<Wg, I, E> {
                 }
             }
             Err(e) => {
-                if let Error::NoIGDGateway = e {
-                    telio_log_info!("Failed to check route: route does not exist")
-                } else {
-                    telio_log_info!("Error: {}", e);
+                telio_log_info!("Error validating endpoint: {}", e);
 
-                    telio_log_info!("Deleting an old Upnp endpoint");
-                    let detete = self.igd_gw.delete_endpoint_routes(
+                telio_log_info!("Dropping Upnp endpoint");
```

Comment:
1. seems to me this should be `telio_log_error!`
2. is there a reason not to merge those two logs into one?

Reply:
1. in that case this sounds like it should be `telio_log_warn!`.
2. imo, lets merge it